docs/Data correction documentary-cleared.ipynb

```diff
@@ -0,0 +1,301 @@
+    "- Note: The readout of which spin-channel to use for $I^{DB,unpol-in,p|a}_{\\lambda, t}$ is irrelevant, as they are the same."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3dc34bc5-06de-48d0-acda-eafb22f09b80",
+   "metadata": {},
+   "source": [
+    "#### 4.3 Spin-dependent transmission values\n",
+    "\n",
+    "Now both $O^{p|a}(\\lambda)=)O^{0, p|a}*\\lambda$ and $P^{3He, p|a}(t)$ are known. <br>\n",
+    "Following Refs (1, 3), the polarization-dependent transmission matrices of a polarized neutron beam through the 3He cells can be computed by inserting \n",
+    "\n",
+    "$\\hat{T}^{\\pm, p|a}(t,\\lambda) = T^g*e^{-O^{0, p|a}*\\lambda\\pm O^{0, p|a}*\\lambda*P^{3He, p|a}(t)}$,\n",
+    "\n",
+    "  \n",
+    "into:\n",
+    "\n",
+    "\\begin{equation}\n",
+    "\\hat{T}^{p}=\\\\\n",
+    "\\begin{pmatrix} \n",
+    "T^{p,+}(t,\\lambda) & 0 & T^{p,-}(t,\\lambda) & 0 \\\\ 0 & T^{p,+}(t,\\lambda) & 0 & T^{p,-}(t,\\lambda) \\\\ T^{p,-}(t,\\lambda) & 0 & T^{p,+}(t,\\lambda) & 0 \\\\ 0 & T^{p,-}(t,\\lambda) & 0 & T^{p,+}(t,\\lambda)  \n",
+    "\\end{pmatrix}\n",
+    "\\end{equation},\n",
+    "\n",
+    "\n",
+    "\\begin{equation}\n",
+    "\\hat{T}^{a}=\\\\\n",
+    "\\begin{pmatrix} \n",
+    "T^{a,+}(t,\\lambda) &  T^{a,-}(t,\\lambda) & 0 &0\\\\ T^{a,-}(t,\\lambda) &  T^{a,+}(t,\\lambda) & 0 & 0\\\\ 0&0& T^{a,+}(t,\\lambda) &  T^{a,-}(t,\\lambda) \\\\ 0&0& T^{a,-}(t,\\lambda) &  T^{a,+}(t,\\lambda)\n",
+    "\\end{pmatrix}\n",
+    "\\end{equation}, \n",
+    "\n",
+    "- Note: same function for both cells, but with different input parameters\n",
+    "- Note: in principle the incoming beam to the polarizer is unpolarized, so $\\hat{T}^{unpol-in}(\\lambda, t)$ isntead of $\\hat{T}^{\\pm, p|a}(t,\\lambda)$ could be used. However, the results will be the same in this case without the need to differ between the cases."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8a1aee42-2dab-438a-aef4-6d90f1818038",
+   "metadata": {},
+   "source": [
+    "4.4 Sample data correction\n",
+    "\n",
+    "For polarizing $^3$He-cells without neutron flippers in the beamline, the correction matrix $\\hat{M}$  becomes $\\hat{M} = \\hat{T}^{a}\\hat{T}^{p}$. To correct the measured data I, we need to apply the inverse matrix to the data ($inv(\\hat{M})$):\n",
+    "\n",
+    "\\begin{equation}\n",
+    "\\begin{pmatrix} I^{++}_{t,\\lambda} \\\\ I^{+-}_{t,\\lambda}  \\\\ I^{-+}_{t,\\lambda} \\\\  I^{--}_{t,\\lambda}\\end{pmatrix} = \\hat{T}^a(t,\\lambda)*\\hat{T}^p(t,\\lambda) \\\\\n",
+    "\\begin{pmatrix} S^{++}_{t,\\lambda} \\\\ S^{+-}_{t,\\lambda}  \\\\ S^{-+}_{t,\\lambda} \\\\  S^{--}_{t,\\lambda}\\end{pmatrix}\n",
+    "\\end{equation}\n",
+    "\n",
+    "\\begin{equation}\n",
+    "inv(\\hat{T}^a(t,\\lambda)*\\hat{T}^p(t,\\lambda)) \\begin{pmatrix} I^{++}_{t,\\lambda} \\\\ I^{+-}_{t,\\lambda}  \\\\ I^{-+}_{t,\\lambda} \\\\  I^{--}_{t,\\lambda}\\end{pmatrix} = \\\\\n",
```

Comment:
`inf` is not a math operator. Use `$^-1$ (probably combined with `left(` and `\right)$`)?

Reply:
looks good like that!